go-controller/pkg/kubevirt/pod.go

```diff
@@ -123,11 +123,14 @@ func EnsurePodAnnotationForVM(watchFactory *factory.WatchFactory, kube *kube.Kub
 	return podAnnotation, nil
 }
 
-// IsMigratedSourcePodStale return true if there are other pods related to
-// to it and any of them has newer creation timestamp.
+// IsMigratedSourcePodStale return false is the pod is live migratable and not completed
```

Comment:
`IsMigratedSourcePodStale return false if the pod is live migratable, not completed and is the running VM pod with newest creation timestamp`